lrtable/src/lib/statetable.rs

```diff
@@ -428,14 +464,9 @@ where
         }
     }
 
-    /// Return a list of shift/reduce conflicts.
-    pub fn sr_conflicts(&self) -> &Vec<(TIdx<StorageT>, PIdx<StorageT>, StIdx)> {
-        &self.shift_reduce
-    }
-
-    /// Return a list of reduce/reduce conflicts.
-    pub fn rr_conflicts(&self) -> &Vec<(PIdx<StorageT>, PIdx<StorageT>, StIdx)> {
-        &self.reduce_reduce
+    /// Return a struct containing all conflicts or None if there aren't any.
+    pub fn conflicts(&self) -> &Option<Conflicts<StorageT>> {
```

Comment:
It might be better to make this `Option<&Conflicts<...>>` to give yourself some flexibility in the future. You'd need to change the body to `self.conflicts.as_ref()`.